docs/configuration/version.md

```diff
@@ -425,3 +425,8 @@ property:
     scmVersion {
         sanitizeVersion.set(false)
     }
+
+## Troubleshooting
+In case of problems you can try to use verbose mode:
+
+    #./gradlew release -Prelease.verbose
```

Comment:
See comment bellow. We should probably do just `./gradlew [command] -d`, no need for passing custom flag https://docs.gradle.org/current/userguide/logging.html